mods/gen2/items.js

```diff
@@ -30,6 +30,7 @@ exports.BattleItems = {
 				return 3;
 			}
 		},
+		desc: "If holder is a Chansey, it overrides all crit boosting effects.",
```

Comment:
That's not even the main thing Lucky Punch does.

It sets Chansey's crit ratio to 3.